Fixes Pest/Weed Killer so that they don't act like hypertoxins, adds organic weedkiller to compensate.

ArcaneMusic opened the pull request:
Yes ATH I'm on it
## About The Pull Request

Turns out, Pest Killer and Weed Killer both inherited the toxin hydroponics reaction, and in the case of weed killer, it inherited both Toxin and Plant-B-Gone, making it hilariously effective at murdering your plant and leaving behind no survivors. Very easy fix,

## Why It's Good For The Game

Unexpected interaction that's been plaguing hydroponics for a bit now, and enables weed killer to do it's thing without being the most caustic plant killer in existance.

## Changelog
:cl:
add: Organic Pest Killer is now available in the biogenerator.
fix: Pest Killer and Weed Killer now applies the correct amount of toxicity and doesn't individually harm the plant's health.
tweak: Pest/Week killer bottles now have a transfer rate of 1 ,2, and 5 units per use, and a max capacity of 30 units.
/:cl:

ArcaneMusic committed:
Initial Commit.

ATH1909 commented:
don't you still need the 
```
    if(!mytray)
        return
```
check from the generic/default reagent on_hydroponics_apply() proc?

ArcaneMusic committed:
Fineeeeeee, added sanity check.

ExcessiveUseOfCobblestone commented:
I never had an issue with this, I just converted the pest killer to bottles.

I would prefer the spray either came in a different container or the bottle only sprays 1u or something.

Not being able to dump pesticide on your plants or it can end up poisoning the tray was a nice touch (even unintended). Maybe even have a version that doesn't harm in the biogen?

ArcaneMusic committed:
Alright, nerfed according to corn on the cobby

ArcaneMusic commented:
Alright, I adjusted according to your feedback, there's now an organic, bio-generator only weedkiller that lacks most of the toxin buildup for a pesticide. To compensate, I added back a bit more of the toxin buildup from regular weed and pest-killer. 
Additionally, I dropped pest/weed killer to have a transfer rate of 1, 2, or 5, but an overall lower capacity of 30.
